azure/templates/teams-alert-setup.yml

```diff
@@ -0,0 +1,14 @@
+steps:
+  - template: "azure/components/aws-assume-role.yml@common"
+    parameters:
+      role: "auto-ops"
+      profile: "apm_ptl"
+
+  - template: "azure/components/get-aws-secrets-and-ssm-params.yml@common"
+    parameters:
+      secret_ids:
+      - ptl/azure-devops/communications-manager/ALERTS_DEV_API_WEBHOOK_URI
```

Comment:
The value extracted here should be defined with a parameter instead of being hardcoded - this gives us the ability to add more webhook secret values without having to update the pipeline definition.

Reply:
Updated and moved the logic that checks what webhook uri we're using from the teams alert to the teams alert setup file